config/zones/ES.yaml

```diff
@@ -1,7 +1,6 @@
 _comment: "Hydro capacity from IRENA includes pumped storage generation capacity.
   Note: ENTSO-E (not used for hydro capacity) lists 5645 MW for pumped storage CONSUMPTION
-  capacity, whereas GENERATION from pumped storage is included in 'hydro reservoir'
-  category."
```

Comment:
Why was this removed? Has this changed?

Reply:
Pumped hydro generation used to be consolidated with hydro but now the live generation sources splits it up.